torch/fx/graph_module.py

```diff
@@ -124,10 +124,12 @@ def __reduce__(self):
     # we need to define deepcopy otherwise it will call __reduce__
     # and cause symbolic tracing to occur every time we try to copy the object
     def __deepcopy__(self, memo):
-        the_copy = self.__new__(type(self))
-        the_copy.__dict__ = deepcopy(self.__dict__, memo)
-        return the_copy
+        fake_mod = torch.nn.Module()
+        fake_mod._modules = copy.deepcopy(self._modules)
```

Comment:
This needs to copy all the properties in `self`, it misses attributes, parameters, buffers, etc.

Reply:
```
m = torch.nn.Module()
m.__dict__ = copy.deepcopy(self.__dict__)
return GraphModule(m, self.graph)
```